### PR TITLE
Apply proofread changes

### PR DESCRIPTION
**Pull request by cilim**
@dingo-d molim te provjeri jel sve i dalje ok nakon lektoriranja svih .md fileova. Ako je sve u redu, slobodno mergeaj PR :)

### CODE REVIEW

File: Automated testing in WordPress/Core tests.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ To run unit tests, you need to specify the `phpunit.xml.dist` file. An example m
 
 Here we've specified that code coverage should be created. For that to be done, you need to have Xdebug installed on your system. Keep in mind that Xdebug may significantly reduce the speed of your tests.
 
-In the root directory—next to `wp-tests-config.php`, the `tests/` folder, and the `phpunit.xml.dist` file, run:
+In the root directory—next to `wp-tests-config.php`, the `tests/` folder, and the `phpunit.xml.dist` file—run:
```
Comment (dingo-d) [inline Automated testing in WordPress/Core tests.md]:
```suggestion
In the root directory, where `wp-tests-config.php` file, the `tests/` folder, and the `phpunit.xml.dist` file are located, run:
```

File: Coding standards/PHP coding standards.md [hunk 2]
```diff
@@ -4,37 +4,35 @@ To check code automatically, we use [PHP_CodeSniffer](https://github.com/squizla
...
+  * [Memoization](#memoization)
+  * [Links on functional PHP programming](#links-on-functional-php-programming)
 - [Optimizations](#optimizations)
-  - [Array checking](#array-checking)
-  - [Using array_push() and similar functions](#using-arraypush-and-similar-functions)
-  - [Caching](#caching)
-  - [I18n](#i18n)
-  - [A11y](#a11y)
+  * [Array checking](#array-checking)
+  * [Using array_push() and similar functions](#using-array-push---and-similar-functions)
```
Comment (dingo-d) [inline Coding standards/PHP coding standards.md]: Not sure why this is changed, probably shouldn't have been.

File: Coding standards/PHP coding standards.md [hunk 3]
```diff
@@ -45,7 +43,7 @@ File names should be written in lowercase letters with `-` as a separator betwee
 
 Files containing a class should be named `class-{classname}.php`. There should always be only one class per file.
 
-Use [eightshift-boilerplate](https://github.com/infinum/eightshift-boilerplate) as a basis for any template-based website. You should follow the existing structure of the theme there.
+Use [wp-boilerplate](https://github.com/infinum/wp-boilerplate) as a basis for any template-based website. You should follow the existing structure of the theme there.
```
Comment (dingo-d) [inline Coding standards/PHP coding standards.md]:
```suggestion
Use [eightshift-boilerplate](https://github.com/infinum/eightshift-boilerplate) as a basis for any template-based website. You should follow the existing structure of the theme there.
```

File: Coding standards/PHP coding standards.md [hunk 4]
```diff
@@ -146,19 +144,18 @@ Always add visibility keywords to methods and properties inside classes (`public
...
 ### Inline statements
 
-Inline statements should have opening and closing PHP tags on the same line
+Inline statements should have the opening and closing PHP tags on the same line.
 
 ```php
+<?php
 // Yes:
-<?php echo esc_html( $x ); ?>
+echo esc_html( $x );
```
Comment (dingo-d) [inline Coding standards/PHP coding standards.md]:
```suggestion
<?php echo esc_html( $x ); ?>
```

File: Coding standards/PHP coding standards.md [hunk 5]
```diff
@@ -146,19 +144,18 @@ Always add visibility keywords to methods and properties inside classes (`public
...
 
 // No:
-<?php
-  echo esc_html
-?>
+echo esc_html( $x );
```
Comment (dingo-d) [inline Coding standards/PHP coding standards.md]:
```suggestion
<?php
  echo esc_html	
?>
```

File: Development environment/Docker.md [hunk 6]
```diff
@@ -1,16 +1,137 @@
 In order to install a WordPress development environment on [Docker](https://www.docker.com/), you'll need to install Docker and use [docker-compose](https://docs.docker.com/compose/).
```
Comment (dingo-d) [inline Development environment/Docker.md]: This entire file shouldn't be touched. Maybe some things like `DevOps` or small typos. But the suggestion is an old file.

File: Development environment/Vagrant setup.md [hunk 7]
```diff
@@ -91,13 +91,13 @@ In our case, we connect via Wi-Fi, so choose 1. If you're connecting via Etherne
 
 ### Using Webpack and BrowserSync Plugin
 
-Here at Infinum, we use [eightshift-boilerplate](https://github.com/infinum/eightshift-boilerplate) to kick start our development. It is a modern method that uses [Webpack](https://webpack.js.org/) to bundle your assets.
+Here at Infinum, we use [wp-boilerplate](https://github.com/infinum/wp-boilerplate) to kick-start our development. It is a modern method that uses [Webpack](https://webpack.js.org/) to bundle your assets.
```
Comment (dingo-d) [inline Development environment/Vagrant setup.md]:
```suggestion
Here at Infinum, we use [eightshift-boilerplate](https://github.com/infinum/eightshift-boilerplate) to kick-start our development. It is a modern method that uses [Webpack](https://webpack.js.org/) to bundle your assets.
```

File: Gutenberg/General.md [hunk 8]
```diff
@@ -1,35 +1,34 @@
...
 
 ## Documentation
 We work with our custom-built Eightshift Boilerplate Internal project. You will be working with these libs and should also read their Readme and Wiki materials:
-
-1. [Eightshift Boilerplate](https://github.com/infinum/eightshift-boilerplate)
+1. [Eightshift Boilerplate Internal](https://github.com/infinum/eightshift-boilerplate-internal)
 2. [Eightshift Libs](https://github.com/infinum/eightshift-libs)
-3. [Eightshift Frontend Libs](https://github.com/infinum/eightshift-frontend-libs)
-
+3. [Eightshift Blocks](https://github.com/infinum/eightshift-blocks)
```
Comment (dingo-d) [inline Gutenberg/General.md]: This must be removed. It's not used

File: Plugin development/General.md [hunk 9]
```diff
@@ -1,14 +1,16 @@
-Plugin development doesn't really differ from the theme development. You can use our boilerplate with libs to build a plugin in pretty much the same way you'd build a theme.
+Plugin development is carried out using the [WordPress Plugin Boilerplate](http://wppb.io/). You can find a simple generator for your plugin [here](https://wppb.me/).
```
Comment (dingo-d) [inline Plugin development/General.md]: Plugin development doesn't really differ from the theme development. You can use our boilerplate with libs to build a plugin in pretty much the same way you'd build a theme.
```suggestion
Plugin development doesn't really differ from the theme development. You can use our boilerplate with libs to build a plugin in pretty much the same way you'd build a theme.
```

File: Plugin development/General.md [hunk 10]
```diff
@@ -1,14 +1,16 @@
...
 
-Themes usually handle presentational logic, while plugins handle business logic.
+Note that plugins are written in an object-oriented way, whereas the theme is usually written in a procedural way. Object-oriented programming allows for a better separation of concerns.
+
+We are also working on our version of the plugin boilerplate, which can be found [here](https://github.com/infinum/wp-boilerplate-plugin).
```
Comment (dingo-d) [inline Plugin development/General.md]: This needs to be replaced with:

`Themes usually handle presentational logic, while plugins handle business logic.`

File: Theme development/General.md [hunk 11]
```diff
@@ -1,30 +1,30 @@
...
 
-Here at Infinum, we are always looking to advance our knowledge about modern technologies. This is why we've developed a [eightshift-boilerplate](https://github.com/infinum/eightshift-boilerplate). This is sort of a backbone solution to start your project. It uses [Webpack](https://webpack.js.org/) to bundle your files, [Sass](http://sass-lang.com/) for modern and quick styling using [BEM methodology](http://getbem.com/), [ESLint](http://eslint.org/) for JavaScript check, and [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) for checking PHP files.
+Here at Infinum, we are always looking to advance our knowledge of modern technologies. This is why we've developed a [wp-boilerplate](https://github.com/infinum/wp-boilerplate). This is sort of a backbone solution to start your project. It uses [Webpack](https://webpack.js.org/) to bundle your files, [Sass](http://sass-lang.com/) for modern and quick styling using [BEM methodology](http://getbem.com/), [ESLint](http://eslint.org/) for JavaScript check, and [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) for checking PHP files.
```
Comment (dingo-d) [inline Theme development/General.md]:
```suggestion
Here at Infinum, we are always looking to advance our knowledge of modern technologies. This is why we've developed a [eightshift-boilerplate](https://github.com/infinum/eightshift-boilerplate). This is sort of a backbone solution to start your project. It uses [Webpack](https://webpack.js.org/) to bundle your files, [Sass](http://sass-lang.com/) for modern and quick styling using [BEM methodology](http://getbem.com/), [ESLint](http://eslint.org/) for JavaScript check, and [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) for checking PHP files.
```

File: Theme development/WP boilerplate.md [hunk 12]
```diff
@@ -1,21 +1,21 @@
 Infinum's WordPress boilerplate is the basis of every WordPress-based project we work on at Infinum.
 
-The repository can be found [here](https://github.com/infinum/eightshift-boilerplate).
+The repository can be found [here](https://github.com/infinum/wp-boilerplate).
```
Comment (dingo-d) [inline Theme development/WP boilerplate.md]:
```suggestion
The repository can be found [here](https://github.com/infinum/eightshift-boilerplate).
```

File: Theme development/WP boilerplate.md [hunk 13]
```diff
@@ -1,21 +1,21 @@
...
 
 Read the readme of the project to learn more about other added bonuses.
 
-This is in-depth documentation on how to run and modify the WordPress boilerplate code. The intended use of this boilerplate is to introduce modern front end development tools to WordPress.
+This is in-depth documentation on how to run and modify the WordPress boilerplate code. The intended use of this boilerplate is to introduce modern front-end development tools to WordPress.
 
 ## Getting started
 
-Read the [readme](https://github.com/infinum/eightshift-boilerplate/blob/master/README.md) of the project to get started with theme development.
+Read the [readme](https://github.com/infinum/wp-boilerplate/blob/master/README.md) of the project to get started with theme development.
```
Comment (dingo-d) [inline Theme development/WP boilerplate.md]:
```suggestion
Read the [readme](https://github.com/infinum/eightshift-boilerplate/blob/master/README.md) of the project to get started with theme development.
```

File: Theme development/WP boilerplate.md [hunk 14]
```diff
@@ -1,21 +1,21 @@
...
 
 ## Wiki
 
-Check out the official [wiki page](https://github.com/infinum/eightshift-boilerplate/wiki) for more information about the boilerplate.
+Check out the official [wiki page](https://github.com/infinum/wp-boilerplate/wiki) for more information about the boilerplate.
```
Comment (dingo-d) [inline Theme development/WP boilerplate.md]:
```suggestion
Check out the official [wiki page](https://github.com/infinum/eightshift-boilerplate/wiki) for more information about the boilerplate.
```

**Comment by cilim** [comment]
> @cilim Ostavio sam komentare kaj bi trebalo promijeniti, ostalo je ok.

@dingo-d hoces da ja applyam promjene ili ces ti? :)

**Comment by dingo-d** [comment]
Ajde budem ja :)

**Pull request merged**